packages/ast/src/index.js

```diff
@@ -44,18 +44,34 @@ export function functionNameMetadata(
   };
 }
 
+export function localNameMetadata(
+  value: string,
+  localIndex: number,
+  functionIndex: number
+): LocalNameMetadata {
+  return {
+    type: "LocalNameMetadata",
+    value,
+    localIndex,
+    functionIndex
+  };
+}
+
 export function moduleMetadata(
   sections: Array<SectionMetadata>,
-  functionNames: Array<FunctionNameMetadata>
+  functionNames: Array<FunctionNameMetadata>,
+  localNames: Array<LocalNameMetadata>
 ): ModuleMetadata {
   const n: ModuleMetadata = {
     type: "ModuleMetadata",
     sections
   };
-
   if (functionNames.length) {
     n.functionNames = functionNames;
   }
+  if (localNames.length) {
```

Comment:
While I understand this, I would prefer something more explicit: `localNames.length > 0`.